Producer/Producer.Droid/MainActivity.cs

```diff
@@ -29,6 +31,33 @@ protected override void OnCreate (Bundle savedInstanceState)
 			setupViewPager ();
 		}
 
+		public override bool OnCreateOptionsMenu (Android.Views.IMenu menu)
+		{
+			MenuInflater.Inflate (Resource.Menu.menu_settings, menu);
+			return base.OnCreateOptionsMenu (menu);
+		}
+
+		public override bool OnOptionsItemSelected (IMenuItem item)
+		{
+			switch(item.ItemId)
+			{
+				case Resource.Id.action_settings:
+
+					Toast.MakeText (this, "Settings selected", ToastLength.Short).Show ();
```

Comment:
this toast probably isn't necessary but if it's just for debug throw a comment on it saying the same